<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Run on chrome"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="5.062">
  <properties />
            <testcase name="TC_02_Register_Invalid_Email" time="0.546" classname="com.nopcommerce.user.TC_Register_Extent">
                <failure type="java.lang.AssertionError"
                  message="expected [Wrong email 1] but found [Wrong email]"
              >
<![CDATA[java.lang.AssertionError: expected [Wrong email 1] but found [Wrong email]
  org.testng.Assert.fail(Assert.java:96)
  org.testng.Assert.failNotEquals(Assert.java:776)
  org.testng.Assert.assertEqualsImpl(Assert.java:137)
  org.testng.Assert.assertEquals(Assert.java:118)
  org.testng.Assert.assertEquals(Assert.java:453)
  org.testng.Assert.assertEquals(Assert.java:463)
  com.nopcommerce.user.TC_Register_Extent.TC_02_Register_Invalid_Email(TC_Register_Extent.java:72)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:564)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
  org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
  java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
  java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
  java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
  java.base/java.lang.Thread.run(Thread.java:832)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="TC_03_Register_Sucessfully" time="3.526" classname="com.nopcommerce.user.TC_Register_Extent" />
                <testcase name="TC_01_Register_Empty_Data" time="0.990" classname="com.nopcommerce.user.TC_Register_Extent" />
      
</testsuite>
